LibreNMS/__init__.py

```diff
@@ -106,14 +130,11 @@ def query(self, query, args=None):
         :return: the cursor with results
         """
         try:
-            cursor = self.db_conn().cursor()
-            cursor.execute(query, args)
-            cursor.close()
-            return cursor
+            return self.__query__(query, args)
         except Exception as e:
-            critical("DB Connection exception {}".format(e))
-            self.close()
-            raise
+            critical("ERROR: Mysql databse has problem !{}".format(e))
+            times.sleep(20)
+            return self.__query__(query, args)
```

Comment:
What happens after it is not back after it the sleep(20)?  Looks like it fails the same way.